lib/smart_app_launch/token_refresh_test.rb

```diff
@@ -16,17 +16,18 @@ class TokenRefreshTest < Inferno::Test
       the Pragma response header field with a value of no-cache to be
       consistent with the requirements of the inital access token exchange.
     )
-    input :smart_token_url, :refresh_token, :client_id, :received_scopes
-    input :client_secret, optional: true
+    input :smart_token_url, :refresh_token, :received_scopes
```

Comment:
Token url and refresh token are auth_info fields. Received scopes isn't, but maybe it should be?